mod/deckorate_research/set/type/relationship/jsonld.rb

```diff
@@ -0,0 +1,44 @@
+format :jsonld do
+
+    def molecule
+        answer_jsonld()
+    end
+
+  private
+
+    def answer_jsonld  
+        metric = card.metric_card
+        print ("Metric Cars: #{metric}")
+        {
+            "@context" => "#{request.base_url}/context/#{card.type}.jsonld",
+            "@id"      => path(mark: card.name, format: nil),
+            "@type"    => card.type,
+            "name" => card.name,
+            "metric" => path(mark: card.metric),
+            "subject_company" => path(mark: card.company),
+            "object_company" => path(mark: card.related_company),
+            "predicate" => metric.metric_title,
+            "value" => get_value(metric),
+            "unit" => metric.unit.presence,
+            "year" => card.year,
+            "source" => get_sources, 
+            "license" => license_url(metric)
+
+        }.compact
+    end
+
+    def license_url metric
```

Comment:
all the code below is duplicate, right?

You can either put it in a shared set or an abstract one. Eg Abstract::JsonldSupport...